Azure/azure-cli
# [AppService] BREAKING CHANGE: az webapp create, az webapp up - Update available webapp runtimes

**Description**  
Fixes #15269 

Updates runtimes to match the runtimes listed in Portal when creating webapp (aka.ms/canary).
When creating a webapp, only the updated list of runtimes can be used

Affects following commands:
- `az webapp up`: can only create webapps using the updated list of runtimes
- `az webapp create`: can only create webapps using the updated list of runtimes
- `az webapp list-runtimes`: displays the updated list of runtimes

Windows added:
- Node 10.10, Node 12 LTS

Windows removed:
- Node 10.0, Node 10.15
- Php 7.4
- Java 1.7 Tomcat 7.0, 8.0, 8.5, 9.0
- Java 1.7 Jetty 9.1, 9.3
- Java 1.8 Tomcat 7.0, 8.0
- Java 1.8 Jetty 9.1, 9.3
- Java 11 Tomcat 7.0, 8.0
- Java 11 Jetty 9.1, 9.3

Linux added:
- Node 10.6
- JBoss EAP 7.2

Linux removed:
- Node 10.10
- Node 10.12
- Node 10.16
- Node 12.9

**Testing Guide**  
Run `az webapp list-runtimes` and `az webapp list-runtimes --linux`. The results should be the same as the list of runtimes available to choose from when creating a webapp on canary version of Portal (aka.ms/canary).

Creating webapps using the commands `az webapp up` and `az webapp create` should only be able to create webapps from the updated list of runtimes


---

This checklist is used to make sure that common guidelines for a pull request are followed.
